app/components/Modals/SendModal/ReadCode/ReadCode.jsx

```diff
@@ -11,7 +11,8 @@ import baseStyles from '../SendModal.scss'
 import styles from './ReadCode.scss'
 
 type Props = {
-  gotoNextStep: Function
+  gotoNextStep: Function,
```

Comment:
I think types that are more descriptive than Function are more helpful. E.g. (networkId: string) => string